module/PowerNSX.psm1

```diff
@@ -28537,6 +28542,9 @@ function Set-NsxFirewallRule {
         [Parameter (Mandatory=$false)]
             [ValidateSet("Allow","Deny", "Reject")]
             [string]$action,
+            [Parameter (Mandatory=$false)]
```

Comment:
Can you pull the indentation back on this line to make it the same as the ones above and below